.github/workflows/deploy.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy Gem
+
+on:
+  push:
+    tags:
+      - v*
+
+permissions:
+  contents: read
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+      id-token: write
+
+    steps:
+      # Set up
+      - uses: actions/checkout@v4
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+          ruby-version: ruby
+
+      # Release
+      - uses: rubygems/release-gem@v1
```

Comment:
Does this mean that new versions of the gem will be automatically pushed to rubygems?

Reply:
It is going to call `rake release` https://github.com/rubygems/release-gem/blob/v1/action.yml#L30C12-L30C36